### PR TITLE
Finding module WapQuery

### PR DESCRIPTION
**Pull request by jabolina**
Fix issue #60. I need some review in the code, if you think that this is OK or need any changes.

### CODE REVIEW

**Comment by jabolina** [comment]
Just discovered a better way, I will commit the change latter

**Comment by alfred82santa** [comment]
It looks the real problem is with check (https://github.com/alfred82santa/whalesong/pull/61/files#diff-830019954e86ede7d87934ca0a5df229L58). 

Probably there is no `queryExists` function anymore. We must look for a better identification.

I'm sorry, but I can not accept current solution. It will fail just with a new js compilation.

File: js/src/whalesong/index.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function getArtifactsDefs() {
   return {
     'conn': (module) => module.default && typeof module.default == "object" && 'ref' in module.default && 'refTTL' in module.default ? module.default : null,
     'store': (module) => module.Chat && module.Msg ? module : null,
-    'wap': (module) => module.queryExist ? module : null,
+    'wap': (module) => (module.queryExist) ? module : ((module.default && module.default.queryExist) ? module.default : null),
```
Comment (alfred82santa) [inline js/src/whalesong/index.js]: It looks like it should be:

```module.default.queryExist ?```

**Comment by alfred82santa** [comment]
About QR, I don't have problems, yet. It is possible they are showing different versions depends on country.

File: js/src/whalesong/index.js (hunk 1 above)
Comment (alfred82santa) [reply js/src/whalesong/index.js]: Please, don't abuse of parenthesis...

**Pull request merged**